### ISSUE TITLE
Create example app

### ISSUE THREAD

**Issue by itavero** [itavero/flutter-ble-uart#3]
As mentioned in #1, there is a need for an example app.

The example probably only needs two screens:
1. Scan: discover compatible BLE devices.
2. Terminal: exchange message with a compatible device.

**Comment by bbinet** [comment]
I would also be very interested by a basic example app.

**Comment by zwells** [comment]
https://github.com/BradHacker/cycler-controller/blob/master/lib/main.dart is an excellent example app!

**Comment by bbinet** [comment]
@zwells https://github.com/BradHacker/cycler-controller does not seem to use this flutter-ble-uart lib

**Comment by zwells** [comment]
@bbinet Correct, it contains the functions/logic to use uart over ble with only flutter_blue library.  This library is dependent on a much older version of flutter_blue which has many issues.
